packages/account/src/Sections/Profile/PersonalDetails/constants.ts

```diff
@@ -5,3 +5,10 @@ export const account_opening_reason_list = [
     { text: localize('Income Earning'), value: 'Income Earning' },
     { text: localize('Hedging'), value: 'Hedging' },
 ];
+
+export const salutation_list = [
+    { text: localize('Mr'), value: 'Mr' },
+    { text: localize('Mrs'), value: 'Mrs' },
+    { text: localize('Ms'), value: 'Ms' },
+    { text: localize('Miss'), value: 'Miss' },
```

Comment:
Ms and Miss? :)

Reply:
Will remove this salutation list since we are not using it anywhere

Reply:
How come? They seem like legit salutations to me?